stl/inc/execution

```diff
@@ -4383,13 +4440,14 @@ struct _Static_partitioned_exclusive_scan2 {
     _Static_partition_range<_FwdIt1, _Diff> _Basis1;
     _Static_partition_range<_FwdIt2, _Diff> _Basis2;
     _Parallel_vector<_Scan_decoupled_lookback<_Ty>> _Lookback;
+    _Scan_intermediate_storage<_Ty, _FwdIt2> _Intermediate_result;

```

Comment:
# :warning:

This is a representation change, so we must rename `struct _Static_partitioned_exclusive_scan2` in order to avoid severe ODR violations. Occurs below.

(As usual, there will still be function-level ODR violations, but those are ones we can get away with.)